p2p/node.go

```diff
@@ -53,6 +53,10 @@ type QriNode struct {
 	// ipfs node provided by repo
 	Repo repo.Repo
 
+	// ProfileService listens for requests for profile exchanges and allows
+	// your node to request profile exchanges
+	profileService *ProfileService
```

Comment:
I'd personally prefer this field be named `profiles`. Unexported fields get to assume the user knows it's a `ProfileService` and will be referring to that API when working. I find it makes method definitions a little easier to read. For example:

```go
n.profileService.IsQriPeer(pi.ID)
```

becomes:
```go
n.profiles.IsQriPeer(pi.ID)
```

I find the latter easier to think about, and it makes me think more about the `ProfileService` API by creating a distinct use case.

Reply:
I'm planning on doing a larger rename in the next iteration, since it is going to grow past just a profile service and into more of a qri identification/qri connection manager type service. qriIDService? qriConns?

Happy to do this rename now tho.